src/handlers/issue-deduplication.ts

```diff
@@ -110,32 +115,97 @@ async function handleSimilarIssuesComment(
         { issueNodeId: issue.issue_id }
       );
       issueUrl.similarity = Math.round(issue.similarity * 100).toString();
+      issueUrl.mostSimilarSentence = findMostSimilarSentence(context.payload.issue.body || "", issueUrl.node.body);
       return issueUrl;
     })
   );
 
-  let finalIndex = 0;
-  const commentBody = issueList
-    .filter((issue) =>
-      matchRepoOrgToSimilarIssueRepoOrg(payload.repository.owner.login, issue.node.repository.owner.login, payload.repository.name, issue.node.repository.name)
-    )
-    .map((issue, index) => {
-      const modifiedUrl = issue.node.url.replace("https://github.com", "https://www.github.com");
-      return `[^0${index + 1}^]: [${issue.node.title}](${modifiedUrl}) ${issue.similarity}%`;
-    })
-    .join("\n");
+  const relevantIssues = issueList.filter((issue) =>
+    matchRepoOrgToSimilarIssueRepoOrg(payload.repository.owner.login, issue.node.repository.owner.login, payload.repository.name, issue.node.repository.name)
+  );
 
-  if (commentBody.length === 0) {
+  if (relevantIssues.length === 0) {
     return;
   }
-  const footnoteLinks = [...Array(++finalIndex).keys()].map((i) => `[^0${i + 1}^]`).join("");
-  const body = "\n###### Similar " + footnoteLinks + "\n\n" + commentBody;
 
-  //Append the new foot note
+  const issueBody = context.payload.issue.body || "";
```

Comment:
Instead of these empty strings it should probably throw an error. If there's no spec how can it be similar? 

This is a great example why empty strings are banned in our code. Don't use them.